v2/en/events.md

```diff
@@ -0,0 +1,15 @@
+<!---
+    @title         Events
+    @creator       Yichun Zhang
+    @modifier      Yichun Zhang
+    @created       2016-11-28 15:58 GMT
+    @changes       1
+--->
+
+
+#  English Events
+
+#  Chinese Events
+* OpenResty Con 2016 (2016-12-10 9:00M at Shenzhen, China)
```

Comment:
@agentzh  Thx for point this error, i will remember this point about Internationalization date time :grin: